src/nft/components/explore/CarouselCard.tsx

```diff
@@ -161,12 +163,45 @@ const TableElement = styled.div`
   line-height: 20px;
 `
 
+interface MarketplaceRowProps {
+  marketplace: string
+  floor?: string
+  listings?: string
+}
+
+export const MarketplaceRow = ({ marketplace, floor, listings }: MarketplaceRowProps) => {
+  return (
+    <>
+      <TableElement>{marketplace}</TableElement>
+      <TableElement>{floor ?? '-'}</TableElement>
+      <TableElement>{listings ?? '-'}</TableElement>
+    </>
+  )
+}
+
 interface CarouselCardProps {
   collection: TrendingCollection
   onClick: () => void
 }
 
+const MARKETS_TO_CHECK = [Markets.Opensea, Markets.X2Y2, Markets.LooksRare] as const
```

Comment:
Can this be replaced with `Object.keys(MARKETS_ENUM_TO_NAME)`?

Reply:
tried unfortunately did not work